src/test/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImplTest.scala

```diff
@@ -146,7 +150,7 @@ class OfferProcessorImplTest extends UnitTest {
           val taskId = Task.Id(taskInfo.getTaskId)
           val launch = f.launchWithNewTask(
             taskInfo,
-            InstanceUpdateOperation.Provision(dummyInstance.provisioned(AgentInfoPlaceholder(), NetworkInfoPlaceholder(), AppDefinition(appId), clock.now(), taskId)),
+            InstanceUpdateOperation.Provision(dummyInstance.instanceId, AgentInfoPlaceholder(), AppDefinition(appId).version, Seq(Task.provisioned(taskId, NetworkInfoPlaceholder(), AppDefinition(appId).version, clock.now())), clock.now()),
```

Comment:
Why do we have `AppDefinition(appId).version` twice? This is hard to read. `AppDefinition(appId)` is used so often that it probably helps to save it as a `val app`.